aiohttp/web_urldispatcher.py

```diff
@@ -131,6 +140,15 @@ def handler(self):
     def name(self):
         """Optional route's name, always equals to resource's name."""
 
+    @property
+    @abc.abstractmethod
+    def canonical(self):
```

Comment:
Should we have `route.canonical`?
`route.resource.canonical` should be enough.
`route.name` exists for historical reasons, routes were added before the resource invention.

Reply:
The issue was asking for a route property, that's why I added it. 

I will change it and I will update the docs (I completely forgot them...)